Assets/HoloToolkit/Input/Scripts/Utilities/MicrophoneHelperSample.cs

```diff
@@ -1,159 +1,34 @@
-            {
-                // The user has turned off access to the microphone. If this occurs, we should show an error, or disable
-                // functionality within the app to ensure that further exceptions aren't generated when 
-                // recognition is attempted.
-                return MicrophoneStatus.MicrophoneUseNotAuthorized;
-            }
-            catch (Exception exception)
-            {
-                // This can be replicated by using remote desktop to a system, but not redirecting the microphone input.
-                // Can also occur if using the virtual machine console tool to access a VM instead of using remote desktop.
-                // You can also get here if you disable all audio recording endpoints
-                if (exception.HResult == NoCaptureDevicesHResult)
-                {
-                    return MicrophoneStatus.MicrophoneNotPresent;
-                }
-                else
-                {
-                    return MicrophoneStatus.MicrophoneUseFailed;
-                }
-            }
-
-            // if microphone is muted, speech recognition will not hear anything.
-            // return appropriate value to indicate that.
-            if(capture.AudioDeviceController.Muted)
-            {
-                return MicrophoneStatus.MicrophoneMuted;
-            }
-
-            return MicrophoneStatus.MicrophoneReady;
-        }
-
-        /// <summary>
-        /// This function provides an easier query when one is only interested to know if the microphone
-        /// can be used or not, without dealing with the specifics of why it may not be working
-        /// </summary>
-        /// <returns>
-        /// Returns true if the microphone is accessible and ready for use.
-        /// </returns>
-        public static async Task<bool> IsMicrophoneReady()
-        {
-            var status = await GetMicrophoneStatus();
-            return status == MicrophoneStatus.MicrophoneReady;
-        }
-#else
-        // if we run in the editor return that microphone is ready by default
-        // It could be useful to test in the editor the behavior where the
-        // microphone is reported ready.
-        public static MicrophoneStatus GetMicrophoneStatus()
-        {
-            return MicrophoneStatus.MicrophoneReady;
-        }
-
-        public static bool IsMicrophoneReady()
-        {
-            var status = GetMicrophoneStatus();
-            return status == MicrophoneStatus.MicrophoneReady;
-        }
-#endif
-    }
-}
-
-public class MicrophoneHelperSample : MonoBehaviour
+namespace HoloToolkit.Unity
 {
-    [Tooltip("Assign key to quickly test the microphone helper code")]
-    public KeyCode checkMicrophoneKey = KeyCode.M;
+    public class MicrophoneHelperSample : MonoBehaviour
+    {
+        [Tooltip("Assign key to quickly test the microphone helper code")]
+        public KeyCode checkMicrophoneKey = KeyCode.M;
```

Comment:
Have we checked to make sure `KeyCode.M` isn't used elsewhere in the project?

Reply:
A search in all .cs files did not return any other references.